tests/compute/nonuniformres-array-of-textures.slang

```diff
@@ -0,0 +1,40 @@
+//TEST:SIMPLE(filecheck=CHECK1):-target spirv -entry main -stage compute
+//TEST:SIMPLE(filecheck=CHECK2):-target spirv -entry main -stage compute -emit-spirv-directly
+
+Texture2D<float> textures[2];
+RWTexture2D<float2> outputTexture;
+uint2 iVals;
+float2 fVals;
+[numthreads(1, 1, 1)]
+void main(uint2 pixelIndex : SV_DispatchThreadID)
+{
+
+    // CHECK1-DAG: OpDecorate %[[VAR1:[a-zA-Z0-9_]+]] NonUniform
+    // CHECK1-DAG: OpDecorate %[[VAR2:[a-zA-Z0-9_]+]] NonUniform
+    // CHECK1-DAG: OpDecorate %[[VAR3:[a-zA-Z0-9_]+]] NonUniform
+    // CHECK1-DAG: OpDecorate %[[VAR4:[a-zA-Z0-9_]+]] NonUniform
+    // CHECK1-DAG: OpDecorate %[[VAR5:[a-zA-Z0-9_]+]] NonUniform
+    // CHECK1-DAG: OpDecorate %[[VAR6:[a-zA-Z0-9_]+]] NonUniform
```

Comment:
I don't think you need "-DAG" here when the result is deterministic in a certain order.

Reply:
let me try to remove it